File: v2/snippets/csharp/common/SampleProgram/SampleProgram.cs
namespace SampleProgram;
// <Import>
using Javonet.Netcore.Sdk;
// </Import>
class SampleProgram
{
	static void Main(string[] args)
	{
		// <Activation>
		Javonet.Activate("your-email", "your-license-key");
		// </Activation>



		var pythonRuntime = Javonet.InMemory().Python();
	}
}

File: v2/snippets/ruby/common/sampleProgram.rb
# <Import>
require 'javonet-ruby-sdk'
# </Import>

# <WholeSnippet>
# <Activation>
Javonet.activate("your-email", "your-license-key")
# </Activation>

# <RuntimeContextCreation>
python_runtime = Javonet.in_memory.python
# </RuntimeContextCreation>

# <GetType>
python_type = python_runtime.get_type("math").execute
# </GetType>

# <GetStaticField>
response = python_type.get_static_field("pi").execute
# </GetStaticField>

# <GetValue>
result = response.get_value
puts result
# </GetValue>	
# </WholeSnippet>